bindata/network/multus/multus.yaml

```diff
@@ -338,24 +418,10 @@ spec:
         - name: WHEREABOUTS_NAMESPACE
           value: "openshift-multus"
       containers:
-      - name: kube-multus
+      - name: kube-multus-additional-cni-plugins
         image: {{.MultusImage}}
-        command: ["/entrypoint.sh"]
-        args:
-        - "--multus-conf-file=auto"
-        - "--multus-autoconfig-dir=/host/var/run/multus/cni/net.d"
-        - "--multus-kubeconfig-file-host=/etc/kubernetes/cni/net.d/multus.d/multus.kubeconfig"
-{{- if eq .DefaultNetworkType "OpenShiftSDN"}}
-        - "--readiness-indicator-file=/var/run/multus/cni/net.d/80-openshift-network.conf"
-{{- else if eq .DefaultNetworkType "OVNKubernetes"}}
-        - "--readiness-indicator-file=/var/run/multus/cni/net.d/10-ovn-kubernetes.conf"
-{{- end}}
-        - "--cleanup-config-on-exit=true"
-        - "--namespace-isolation=true"
-        - "--multus-log-level=verbose"
-        - "--cni-version=0.3.1"
-        - "--additional-bin-dir=/opt/multus/bin"
-        - "--skip-multus-binary-copy=true"
+        command: [ "/bin/bash", "-c", "--" ]
+        args: [ "trap : TERM INT; sleep infinity & wait" ]
         resources:
           requests:
             cpu: 10m
```

Comment:
down below, now that this is just a sleep, it can probably have it's memory request dropped below `150Mi` ;)

Reply:
Good call, got it at `10Mi`